MatrixKit/Models/Room/MXKRoomBubbleComponent.h

```diff
@@ -18,6 +18,8 @@
 
 #import "MXKEventFormatter.h"
 
+@class URLPreviewData;
```

Comment:
Initially I defined the property below as `id`, which didn't seem great. Realised I could do this with the type from Element instead, which seems better to me but pointing it out in case it's a bad idea.

Reply:
We can define a protocol in the kit like `URLPreviewDataProtocol` with the same interface as URLPreviewData from Element. In this case Kit clients will be able to provide their own implementation or reuse the Element one.